enterprise/validators.py

```diff
@@ -0,0 +1,24 @@
+"""
+Database models field validators.
+"""
+import os
+from django.core.exceptions import ValidationError
+from django.utils.translation import ugettext_lazy as _
+
+
+def validate_image_extension(value):
+    """
+    Validate that a particular image extension.
+    """
+    ext = os.path.splitext(value.name)[1]  # [0] returns path+filename
+    valid_extensions = ['.png', ]
```

Comment:
@asadiqbal08 nit: extract allowed extensions into constant?

Bonus-level 1: make constant an overridable setting (and put it into app config in app.py)
Bonus-level 2: using ninja skills, make `logo` field help text pick a list of allowed extensions from that setting.

Bonus levels are **really** optional.